### PR TITLE
[tankerkoenig] Fixed link to 'Tankstellen Finder'

### PR DESCRIPTION
**Pull request by cweitkamp**
- Fixed link to 'Tankstellen Finder'
- Removed URLs from thing / configuration descriptions
- Fixed minor spelling mistakes

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: bundles/org.openhab.binding.tankerkoenig/README.md [hunk 1]
```diff
@@ -16,18 +16,19 @@ While using the mode Opening-Times the channel "station_open" will NOT show "clo
...
+- a personal API-Key
 
-Request a free Tankerkönig API key from: <https://creativecommons.tankerkoenig.de/>  Select the tab "API-Key".
+Request a free Tankerkönig API key from: <https://creativecommons.tankerkoenig.de/> (Select the tab "API-Key").
 
--LocationIDs of the selected gas stations
+- LocationIDs of the selected gas stations
 
-Search for the gas station IDs here: <https://creativecommons.tankerkoenig.de/configurator/index.html>
+Search for the gas station IDs vi the [finder tool](https://creativecommons.tankerkoenig.de/TankstellenFinder/index.html) (Select tab "Tools" -> "Tankstellenfinder").
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.tankerkoenig/README.md]:
```suggestion
Search for the gas station IDs via the [finder tool](https://creativecommons.tankerkoenig.de/TankstellenFinder/index.html) (Select tab "Tools" -> "Tankstellenfinder").
```

**Comment by J-N-K** [comment]
Small patch exception for commit 2.

**Comment by JueBag** [comment]
Thanks for those fixes.
Shouldn't the authors get notified about such, or does this apply only for issues?

**Comment by cweitkamp** [comment]
> Shouldn't the authors get notified about such, or does this apply only for issues?

Yes, they should. But they need to be a member of the [openHAB / Contributors](https://github.com/orgs/openhab/teams/contributors/members) team which is not true in your case. @kaikreuzer Would you be so kind and add @JueBag to the team? Thanks.

**Comment by kaikreuzer** [comment]
> @kaikreuzer Would you be so kind and add @JueBag to the team?

Sure, done!

**Pull request merged**